### ISSUE TITLE
About Loss Functions and Targeted Attacks

### ISSUE THREAD

**Issue by jimo17** [machanic/SimulatorAttack#9]
Hi and thanks for the open source. I am very interested in your paper. I have some questions.
Excuse me, does the loss function used in training the simulator and the loss function used in the simulator attack have to be the same?
Can I use a targeted attack while training the simulator and an untargeted attack while the simulator is attacking?

**Comment by machanic** [comment]
@jimo17 (1) No, they are different loss functions. The loss function used in training the simulator is the Mean-Square-Error(MSE) loss, the loss function used in attacking is the logits-difference loss or cross-entropy loss.
(2) Sorry, I don't understand what do you mean? an untargeted attack while the simulator is attacking? YES, it can do it.
